source/style-guide/markup/directives/metadata.txt

```diff
@@ -7,18 +7,23 @@ Page Metadata Directives
 meta
 ----
 
-Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags 
-aids in searches both internally and through the Internet. You can add 
-both keywords and descriptions. To learn more, see 
+Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags
+aids in searches both internally and through the Internet. You can add
```

Comment:
```suggestion
aids in searches both internally and on external search engines. You can add
```
A suggestion! 'on the internet' would work as well

Reply:
+1

Reply:
Thanks, I incorporated the content in a rewrite of this intro section since it wasn't explaining the concepts very well.